README.md

```diff
@@ -308,6 +308,46 @@ This should be verified by executing the Pelion E2E python test library tests.
 
 <span class="notes">**Note:** Future version will bring in also firmware update test as part of the minimum test set.</span>.
 
+## Contributing platform support
+
+The contribution of platform support to this repository is restricted to Arm Mbed Partners and Arm Engineering teams. If you’d like to add a custom or community-based platform, please fork this repository and add it into your own account.
+Expectations on contributions:
+
+-	No code changes in `main.cpp`.
+This is a minimal and generic application that’s expected to work on out of the box with all platforms listed in the documentation and [Pelion Quick-start](https://os.mbed.com/guides/connect-device-to-pelion/) guide
+
+- No changes to the hash of `mbed-os.lib ` or `mbed-cloud-client.lib` files.
+These are maintained and updated by the Arm team.
+
+-	No extra files or `.mbedignore` with removal of Mbed OS code.
+
+-	Configuration (required)
+     - `mbed_app.json` to add components or features. Please follow the guidelines in the porting section of the docs.
+
```

Comment:
Also, 

- You agree that the configuration changes contributed are considered Apache 2.0 licensed open source content.